src/components/Content/index.tsx

```diff
@@ -44,48 +43,44 @@ export const Content: React.FC = () => {
   const { data: userSettingsData } = useQuery<GetUserSettingsQuery>(
     GET_USER_SETTINGS
   );
-
+  const hasUsedSpruceBefore =
+    userSettingsData?.userSettings?.useSpruceOptions?.hasUsedSpruceBefore ===
+    false;
   localStorage.setItem("userId", get(data, "user.userId", ""));
 
-  const hasUsedSpruceBefore = get(
-    userSettingsData,
-    "userSettings.useSpruceOptions.hasUsedSpruceBefore",
-    true
-  );
   useAnalyticsAttributes();
-  if (!isAuthenticated && initialLoad) {
+
+  if (!isAuthenticated) {
     return <FullPageLoad />;
   }
+
   return (
     <PageLayout>
       <Navbar />
       <ConnectivityBanner />
       <SiteBanner />
       <SlackNotificationBanner />
       <Switch>
-        <PrivateRoute path={routes.task} component={Task} />
```

Comment:
I removed the PrivateRoute component since we have an  `isAuthenticated` condition in this render function body.